webapp/src/main/resources/public/js/components/screenshots/Screenshot.js

```diff
@@ -97,6 +98,7 @@ class Screenshot extends React.Component {
                     <span className="screenshot-glyph">
                         <StatusGlyph status={this.props.screenshot.status} 
                                      onClick={this.props.onStatusGlyphClick}
+                                     disabled={!AuthorityService.canEditScreenshots()}
```

Comment:
see other comments too - can we keep that stateless and with no dependency on AuthorityService

Reply:
Also refactored